whylabs_toolkit/helpers/monitor_helpers.py

```diff
@@ -6,7 +6,7 @@
 
 from whylabs_toolkit.helpers.config import Config
 from whylabs_toolkit.helpers.utils import get_monitor_api, get_models_api
-from whylabs_toolkit.monitor.models import Granularity
```

Comment:
If the models file has a dependency on this file then another option would be to just move the Granularity definition into its own file that they can both use, then you'd keep the type safety.

Reply:
it's true, I think I got a wrong commit in for some reason. fixed now

Reply:
on commit `702868e1852071bc7068f2073ef75d7c643a3ad8` Granularity is not there too